### ISSUE TITLE
Application to join the SemVer team

### ISSUE THREAD

**Issue by gwynne** [semver/semver#987]
Hi, I would like to apply to join the SemVer team, to represent [Swift](https://swift.org) and the [Swift Package Manager](https://www.swift.org/package-manager/) ecosystem. SwiftPM has used SemVer for dependency resolution, including supporting minor and patch version qualifications as well as pre-release identifiers, since its original open source release.

Speaking to my qualifications for such a role, I have extensive experience in Swift and all of its components and tools, including and especially SwiftPM; I'm also part of the Core team of Vapor, the largest server-side Swift framework, for which we actively maintain some 40 interrelated packages. I worked at Apple for almost two years, and before that I spent a number of years working in the PHP community, the CocoaPods ecosystem, the iOS developer community, and several other areas. I've written a [pure-Swift standalone implementation](https://github.com/gwynne/swift-semver) of semantic versioning, and I sit on the [Swift Server Working Group](https://swift.org/sswg), a steering committee comprised of industry members from Apple, Amazon, and other companies and having significant influence over both the Swift on Server ecosystem and Swift itself.

In summary, I think I’d be a valuable addition to the SemVer team, bringing experience and connections which would hopefully benefit everyone.

**Comment by steveklabnik** [comment]
Hey there @gwynne ,

I apologize for taking so long to respond to you. To be frank, I personally am very burned out, and it seems most of the other maintainers are too. The lack of reply here isn't you, it's us. I would be happy to add you, but also, I'm not exactly sure how functional things are right now. @JohnTitor has been doing a tremendous amount of work triaging, but the rest of @semver/maintainers have been MIA for a long time. So, just setting that expectation out there. I would love to get back to revitalizing things around here but it kinda feels impossible.

**Comment by segiddins** [comment]
Thirded.

**Comment by gwynne** [comment]
If this is still possible, I'd love to at least try to contribute what I can! 🙂

**Comment by segiddins** [comment]
@gwynne invite sent!
